P20-P25.s

```diff
@@ -420,7 +420,7 @@ DE-GR-50	TC	2PHSCHNG
 #
 # ABORT MODES:  P23 ABORT IF MARKING SYSTEM OR EXTENDED VERB ACTIVE
```

Comment:
Non-spotted typo.

There should be no space between `P23 ABORT` - i.e `P23ABORT`. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0570.jpg)